include/quicr/quicr_server.h

```diff
@@ -165,7 +181,8 @@ class QuicRServer
    * API for unit test cases .
    */
   QuicRServer(std::shared_ptr<qtransport::ITransport> transport,
-              ServerDelegate& delegate, qtransport::LogHandler& logger);
+              ServerDelegate& delegate /* TODO: Considering shared or weak pointer */,
+              qtransport::LogHandler& logger);
```

Comment:
Should this be called log_handler? Is a LogHandler really a Logger?

Reply:
This PR didn't actually make a change here. It's just formatting.  I'm fine with a name change, but we should do that in a different PR and we should do that for ```quicr_client``` as well.